<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="body" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>830</width>
      <height>530</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>831</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 40pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Runge Kutta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCalculo"/>
   <addaction name="actionTabela"/>
   <addaction name="actionGrafico"/>
   <addaction name="actionSobre"/>
  </widget>
  <action name="actionCalculo">
   <property name="text">
    <string>Calculo</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionTabela">
   <property name="text">
    <string>Tabela</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionGrafico">
   <property name="text">
    <string>Grafico</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
